components/rangeslider/rangesliders/singleHandleSliders/basicRangeslider.js

```diff
@@ -0,0 +1,122 @@
+import SingleHandleRangeSliderBase from './singleHandleRangeSliderBase';
+// eslint-disable-next-line no-unused-vars
+import Rangeslider from '../../script';
+import { clamp, valueToPercent } from '../rangeSliderUtils';
+const SPACE_BETWEEN_GRID_POLS = 10;
+
+/**
+ * This is the basic rangeslider which has single thumb and works with numeric values.
+ */
+export default class BasicRangeSlider extends SingleHandleRangeSliderBase {
+    /**
+     * @param {Rangeslider} rangeslider - The gameface-rangeslider custom element
+     */
+    constructor(rangeslider) {
+        super(rangeslider);
+    }
+
+    /** @inheritdoc */
+    connectedCallback() {
+        this.initSliderState();
+        this.loadTemplate();
+    }
+
+    /** @inheritdoc */
+    updateValueState(value) {
+        value = parseFloat(value);
+
+        if (isNaN(value) || (this.min > value && this.max < value)) {
+            this.value = this.min;
+            return;
+        }
+
+        this.state.value = value;
+        this.updateSliderPositionWithCurrentValue();
+    }
+
+    /** @inheritdoc */
+    getCurrentValuePercent() {
+        return valueToPercent(this.value, this.min, this.max);
+    }
+
+    /** @inheritdoc */
+    updateMinMaxState(name) {
+        this.setMinAndMax();
+        const defaultValue = name === 'min' ? this.min : this.max;
+        this.value = this.min <= this.value && this.max >= this.value ? this.value : defaultValue;
+        this.toggleGrid(this.grid);
+        this.updateSliderPositionWithCurrentValue();
+    }
+
+    /** @inheritdoc */
+    setMinAndMax() {
+        const min = parseFloat(this.rangeslider.getAttribute('min'));
+        const max = parseFloat(this.rangeslider.getAttribute('max'));
+        this.state.min = !isNaN(min) ? min : 0;
+        this.state.max = !isNaN(min) ? max : 100;
+    }
+
+    /** @inheritdoc */
+    setHandleValues() {
+        let valueAttr = this.rangeslider.getAttribute('value');
+
+        valueAttr = parseFloat(valueAttr);
+        this.state.value = !isNaN(valueAttr) ? valueAttr : this.min;
+
+        // checks if the value provided is less than the min or more than the max and sets it to the minimum value
+        this.state.value = this.min <= this.value && this.max >= this.value ? this.value : this.min;
+    }
+
+    /** @inheritdoc */
+    buildGrid() {
+        // calculates the number of pols the grid will have based on the size of the slider
+        const numberOfPols = Math.round(this.wrapper[this.units.offset] / SPACE_BETWEEN_GRID_POLS / 4) * 4; // here we round to a number that is divisible by 4 and to make sure, the last pol has a number
+        const grid = document.createElement('div');
+        grid.classList.add(`guic-${this.rangeslider.orientation}-rangeslider-grid`);
+        for (let i = 0; i <= numberOfPols; i++) {
+            // each forth poll will be larger with a value added

```

Comment:
This is a nice custom attribute candidate :D:D
It would be neat if you could customize the grid :D

Reply:
Good idea. Created a task about that! :D
https://coherent-labs.atlassian.net/browse/COH-18339